packages/framework/tests/Unit/Model/Pricing/RoundingTest.php

```diff
@@ -227,11 +227,9 @@ private function converPricingRoundingTypeToCurrencyRoundingType(int $roundingTy
 
             case 2:
                 return Currency::ROUNDING_TYPE_FIFTIES;
-                break;
```

Comment:
I do not understand this change

Reply:
Because when I call return, all after return will not applied, or not?

Reply:
oh, ok :)